easybuild/easyblocks/h/healpix.py

```diff
@@ -37,17 +37,27 @@
 from easybuild.tools.modules import get_software_root
 from easybuild.tools.run import run_cmd, run_cmd_qa
 from easybuild.tools.systemtools import get_shared_lib_ext
-from distutils.version import LooseVersion
+from easybuild.framework.easyconfig import CUSTOM
```

Comment:
Please keep this sorted alphabetically.

Reply:
TIL: There in Pycharm, it is not enough to click on 

> code → reformat

but one has to click on 

> Code → Optimize Imports

to be fully PEP8 compliant.